src/modules/Shared/components/NoData/NoData.tsx

```diff
@@ -0,0 +1,17 @@
+import { Box } from "@mui/material";
+import Nodata from "../../../../assets/images/nodata.jpg";
+
+export default function NoData() {
+  return (
+    <>
+      
+      <Box sx={{px:4,textAlign:"center" }} >
+  
+        <Box component="img" src={Nodata} alt="" sx={{width:{xs:"80%",md:"35%"}}} />
+        <h2 style={{color: "#1F263E"}}>
```

Comment:
A typography element should be used instead. This will enable us to use colors from the design system directly, and when changing the theme, the typography's styles will change as well.

Reply:
Done